lms/templates/video.html

```diff
@@ -54,33 +54,70 @@ <h3 class="hd hd-4 downloads-heading sr" id="video-download-transcripts_${id}">$
         % if download_video_link or public_sharing_enabled:
             <div class="wrapper-download-video">
                 <h4 class="hd hd-5">${_('Video')}</h4>
+                % if download_video_link:
+                    <span class="icon fa fa-download" aria-hidden="true"></span>
+                    <a class="btn-link video-sources video-download-button" href="${download_video_link}">
+                        ${_('Download video file')}
+                    </a>
+                % endif
+                % if download_video_link and public_sharing_enabled:
+                    <br>
+                % endif
                 % if sharing_sites_info:
                     <div class="wrapper-social-share">
-                        <span class="icon fa fa-share-alt" aria-hidden="true"></span>
-                        ${_('Share on:')}
-                        % for sharing_site_info in sharing_sites_info:
+                        <button
+                            style="background-image: none; background-color: rgb(0, 38, 43); border-radius: 0px; color: white" 
+                            class="social-toggle-btn btn"
+                        >
+                            <span class="icon fa fa-share-alt mr-2" style="text-shadow: none"></span>
+                            ${_('Share this video')}
+                        </button>
+                        <div
+                            hidden
+                            class="container-social-share color-black p-2"
+                            style="width: 300px; border-radius: 6px; background-color: white; box-shadow: 0 .5rem 1rem rgba(0,0,0,.15),0 .25rem .625rem rgba(0,0,0,.15)"
```

Comment:
This and further styling should be in SCSS, no?

Reply:
In theory yes, but in platform, not necessarily.  SCSS requires extra build steps and separates it much further from the code because of the framework, so this type of css is done inline in a number of places.

Reply:
I mean, all the rest of the styling is in SCSS, I would still argue for that wherever possible.